src/components/RecentSearchCard/styles.ts

```diff
@@ -31,7 +31,7 @@ const styles = StyleSheet.create({
   searchValueText: {
     color: 'black',
     fontWeight: '400',
```

Comment:
You can remove every attribute here except for `justifyContent: 'center',` because the global styles is handling it already